api/app/Http/Controllers/ChildEntityController.php

```diff
@@ -157,7 +158,7 @@ public function putMany(Request $request, $id)
         $childModels = $this->getChildModel()
             ->hydrateRequestCollection($requestCollection, $existingChildModels);
 
-        $this->getRelation($parent)->saveMany($childModels);
+        $this->saveMany($parent, $childModels);
```

Comment:
Another option (javascript dev talking here) would be to re-bind the returned Relationship saveMany method to a custom closure that calls parent, and defines the method.

This is the js decorator pattern, and I'm not 100% if it is even possible in php.